src/resources/views/components/toggle-theme.blade.php

```diff
@@ -0,0 +1,53 @@
+@php($personalize = $classes())
+
+<div wire:ignore>
+    <button type="button"
+            role="switch"
+            aria-checked="false"
+            x-on:click="darkTheme = !darkTheme"
+            @if (!$simple)
+            x-bind:class="{
```

Comment:
I think we can save some lines by joining this to the same 9th line, this is quite small.